### PR TITLE
Imx8mp wifi fix

### PR DESCRIPTION
**Pull request by mike-scott**
- move generic nxp89xx feature
- add files to support 8997 PCIE
- enable nxp89xx on imx8mp-lpddr4-evk

### CODE REVIEW

File: meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc [hunk 1]
```diff
@@ -440,6 +440,8 @@ EFI_PROVIDER:imx8mp-lpddr4-evk-ebbr ?= "grub-efi"
 OSTREE_BOOTLOADER:imx8mp-lpddr4-evk-ebbr ?= "grub"
 IMAGE_EFI_BOOT_FILES:sota:imx8mp-lpddr4-evk-ebbr ?= "${@make_efi_dtb_boot_files(d)}"
 WKS_FILE:sota:imx8mp-lpddr4-evk-ebbr ?= "efidisk-sota.wks"
+# imx8mp comes with an AzureWave CM276MA PCIe Card
+MACHINE_FEATURES:append:imx8mm-lpddr4-evk = " nxp89xx mxm-mwifiex-load"
```
Comment (ricardosalveti) [inline meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc:444]: Should be imx8mp instead of imx8mm.

Comment (mike-scott) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc]: Great catch.  I fixed this on my build box and didn't re-sync back to my local files.

Comment (mike-scott) [reply meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc]: Fixed

**Review by MrCry0 — APPROVED** [review]
LGTM

**Review by ricardosalveti — APPROVED** [review]
LGTM

**Pull request merged**